src/Mealz/MealBundle/Service/SlotService.php

```diff
@@ -36,4 +46,24 @@ public function delete(Slot $slot): void
         $this->em->persist($slot);
         $this->em->flush();
     }
+
+    public function getSlotStatusForDay(DateTime $datetime): array
```

Comment:
If we decide to make new SlotService then we should move the related logic from ParticipationService here, i.e. probably no reference to ParticipationService.